Packs/Pcysys/Scripts/PenteraOperationToIncident/PenteraOperationToIncident.yml

```diff
@@ -0,0 +1,27 @@
+commonfields:
+  id: PenteraOperationToIncident
+  version: -1
+name: PenteraOperationToIncident
+script: ''
+type: python
+tags: []
+enabled: true
+args:
+  - name: full_action_report
+    required: true
+    description: Pentera Full Action Report
+    isArray: true
+  - name: custom_fields_output
+    description: Name of the field to serialize details into
+  - name: context_key
```

Comment:
lets add `defaultValue` here

Reply:
not sure what you tried doing here, but i meant to add `defaultValue: PenteraIncidents` for this argument and not `defaultValue: false` to `custom_fields_output`